ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/worldstate/CompleteTaskStep.java

```diff
@@ -17,23 +17,40 @@
 import tech.pegasys.pantheon.metrics.Counter;
 import tech.pegasys.pantheon.metrics.MetricsSystem;
 import tech.pegasys.pantheon.metrics.PantheonMetricCategory;
+import tech.pegasys.pantheon.metrics.RunnableCounter;
 import tech.pegasys.pantheon.services.tasks.Task;
 
-public class CompleteTaskStep {
+import java.text.DecimalFormat;
+import java.util.Objects;
+import java.util.function.LongSupplier;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
+public class CompleteTaskStep {
+  private static final Logger LOG = LogManager.getLogger();
+  private static final int DISPLAY_PROGRESS_STEP = 1000;
```

Comment:
This is logging too often.  Needs to be at least 100,000 maybe even a million.  We request up to 3,840 node at a time (10 concurrent requests for 384 nodes).

Reply:
A million is too rare - I'd go for 100,000.

Reply:
Done.